scripts/sizeSnapshot/webpack.config.js

```diff
@@ -75,6 +75,14 @@ async function getWebpackEntries() {
       name: '@material-ui/system',
       path: 'packages/material-ui-system/build/esm/index.js',
     },
+    {
+      name: 'createBox',
+      path: 'packages/material-ui-system/build/esm/createBox.js',
+    },
+    {
+      name: 'createStyled',
+      path: 'packages/material-ui-system/build/esm/createStyled.js',
+    },
```

Comment:
```suggestion
    {
      name: 'createStyled',
      path: 'packages/material-ui-system/build/esm/createStyled.js',
    },
    {
      name: 'createTheme',
      path: 'packages/material-ui-system/build/esm/createTheme.js',
    },
```

?

Reply:
I didn't measure this one as it's already accounted for inside `createStyled` and `createBox`.